content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -129,6 +129,58 @@ the [kubeadm config migrate](/docs/reference/setup-tools/kubeadm/kubeadm-config/
 For more information on the fields and usage of the configuration you can navigate to our
 [API reference page](/docs/reference/config-api/kubeadm-config.v1beta3/).
 
+### Using kubeadm init with feature gates {#feature-gates}
+
+Kubeadm support a set of feature gates that are unique to kubeadm and can only be applied
+during cluster creation with `kubeadm init`. These features can control the behavior
+of the cluster. Feature gates are removed after a feature graduates to GA.
+
+To pass a feature gate you can either use the `--feature-gates` flag for
+`kubeadm init`, or you can add items into the `featureGates` field when you pass
+a [configuration file](/docs/reference/config-api/kubeadm-config.v1beta3/#kubeadm-k8s-io-v1beta3-ClusterConfiguration)
+using `--config`.
+
+Passing [feature gates for core Kubernetes components](/docs/reference/command-line-tools-reference/feature-gates)
+directly to kubeadm is not supported. Instead, it is possible to pass them by
+[Customizing components with the kubeadm API](/docs/setup/production-environment/tools/kubeadm/control-plane-flags/).
+
+List of feature gates:
+
+{{< table caption="kubeadm feature gates" >}}
+Feature | Default | Stage | Since
+:-------|:--------|:------|:------
+`PublicKeysECDSA` | `false` | Alpha | 1.19
+`RootlessControlPlane` | `false` | Alpha | 1.22
+`UnversionedKubeletConfigMap` | `true` | Beta | 1.23
```

Comment:
UnversionedKubeletConfigMap
- 1.23 alpha
- 1.24 beta

Reply:
it might be better to make the table:

name | value | alpha | beta
------|-------|-------|-----
`foo` | `false` | 1.x | 1.y